django/contrib/auth/management/__init__.py

```diff
@@ -66,7 +66,7 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
     except LookupError:
         return
 
-    if not router.allow_migrate(using, Permission):
+    if not router.allow_migrate(using, 'auth', Permission):
```

Comment:
IMO the reason `app_label` wasn't there in a first place is because routing wasn't supported for `RunPython` and `RunSQL`.

It's not directly in reaction to another change, I thought about this when I addressed https://code.djangoproject.com/ticket/22583 but I tried to be as unobstructive as possible at the time.

Considering that `app_label` is the most common attribute for allow_migrate routing (at least in our docs and tests), that having direct access to it works across every operations, and that it allows for simpler routing code (https://github.com/django/django/pull/4152/files#diff-bf01402cfbc5a9c005f4352c82d94423R310) it's IMO the way to go.